specification/transport/udp/udp.tex

```diff
@@ -0,0 +1,337 @@
+\section{Cyphal/UDP}\label{sec:transport_can}
+
+\hyphenation{Cyphal/UDP}  % Disable hyphenation.
+
+\subsection{Overview}
+
+This section specifies a concrete transport based on the UDP/IPv4 protocol\footnote{%
+    Support for IPv6 may appear in future versions of this specification.
+}, as specified in IETF RFC~768.
+\textbf{
+    As of this version, the Cyphal/UDP specification remains experimental.
+    Breaking changes affecting wire compatibility are possible.
+}
+
+Cyphal/UDP is intended for low-latency, high-throughput intravehicular Ethernet networks with complex topologies,
```

Comment:
It's possible, but wouldn't it sound a bit defensive? The fact that it is not an encapsulation becomes apparent once you're done reading the first two pages.

Reply:
My thinking is just to break a confirmation bias where a reader is starting on this section expecting to find out how to encapsulate Cyphal/CAN over UDP (i.e. the meta-transport types). Rather than defensive, I'd see it as a kindness to avoid confusion in such cases. Something like "dear reader - if you want to encapsulate CAN over UDP you can use meta-transport.can over Cyphal/UDP, which is a first-class IP protocol..." etc.